src/components/hooks/__mocks__/useActivityStream.js

```diff
@@ -1,6 +1,3 @@
 export default function useActivityStream(ID, adapter) {
-  const [roomID] = Object.keys(adapter.datasource);
-  const rooms = adapter.datasource;
-
-  return ID === roomID ? rooms[`${ID}-activities`] : [];
+  return `${ID}-activities` in adapter.datasource ? adapter.datasource[`${ID}-activities`] : [];
```

Comment:
can you make changes for all other mocks then? if you are planning to use `in`? `usePerson`, `useRoom` are all using `includes` right now.